index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="toolbar">
+            <a href="#" id="welcomeText">
+                Welcome
+            </a>
+            <div>
+                <img src="src/header/headerButton.svg" alt="">
+            </div>
+        </nav>
+    </header>
+    <section class="introduction">
+        
+            <h1>
+                Hey there. I’m Fabrizio 
+            </h1>
+            <!--Maybe I can use the id to give it a cool effect later-->
+            <h1 id="secondMainSentence">
+                I’m a software developer
+            </h1>
```

Comment:
For SEO best practices is better only use `h1` once per HTML file, it would be better for your skills to change one of the `h1` tags you use twice to `h2` or `h3` tags.

- [ ] Remove double `h1` tag and only leave one `h1` tag.